temporal-sdk/src/main/java/io/temporal/activity/ActivityMethod.java

```diff
@@ -25,19 +25,19 @@
 import java.lang.annotation.Target;
 
 /**
- * Indicates that the method is an activity method. This annotation applies only to activity
- * interface methods. Not required. Use it to override default activity type name.
+ * Indicates that the method is an Activity method. This annotation applies only to Activity
+ * interface methods. Not required. Use it to override default Activity type name.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.METHOD)
 public @interface ActivityMethod {
 
   /**
-   * Name of the activity type. Default is method name. Also consider using {@link
+   * Name of the Activity type. Default is method name. Also consider using {@link
    * ActivityInterface#namePrefix()}. Note that the prefix is ignored if the name is specified.
```

Comment:
> 2. Let's come up with some rules/standard for when a comment sentence goes to a new line vs starting a new sentence on the same line and having it broken up by a newline.
Sounds good - perhaps stick to same line only if both sentences can fit in one line and not multiple? Else, always have a new line? Like this:
<img width="848" alt="Screen Shot 2021-05-01 at 11 09 58 PM" src="https://user-images.githubusercontent.com/1143334/116804269-5b818100-aad2-11eb-91ed-9a3c4cfbc4b1.png">


> 4. Let's find out the requirements for links. I see them in a bunch of different formats and I want to standardize on one.
So [this](https://stackoverflow.com/a/5916021) best explains why they're different in the current state of things. It's basically because some classes are imported in the class already, therefore they do not need the package name defined in this form `{@link com.xxx.yyy.Class}`. 
I prefer not importing a class just because we need it for documentation purposes and to stick to the `{@link com.xxx.yyy.Class}` convention instead. 

Another thing I noticed was that even if the package is defined, it's not printed in the Javadocs page itself. For example `{@link io.temporal.client.ActivityCompletionClient#complete(byte[], Object)}` prints [this](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/activity/ActivityInfo.html).

Might be counterproductive to have a convention for this case tbh as it's based on the code. If anything it should be for the Javadocs readers, but it looks like it 'works'.  


> 6. Let's standardize on using
> 
> ```
> /**
>  * Comment
>  */
> ```
> 
> vs
> 
> ```
> /** Comment */
> ```
> 
> For all Javadoc comments.
I'm cool with standardizing, but I think the former is for multi-line comments and the latter may be used for single-line comments (it's slightly cleaner to look at). This is mentioned in the Google Style Guide too https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line.